<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>1366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="rec_label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 20pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;가장 많이 주문한 메뉴&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="rec_label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 20pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;최근 주문 내역&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>570</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPlainTextEdit" name="dg_most">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="rec_label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>870</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 18pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;해당 정보가&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;고객님의 정보가 맞습니까?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="rec_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 38pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>단골손님 확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="rec_label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 15pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;전화번호&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;뒷자리 &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="rec_label_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;Agency FB&quot;;
font: 81 15pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>님의 주문내역</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dg_number">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>161</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>391</width>
     <height>50</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>980</y>
     <width>619</width>
     <height>203</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="btn_dg_yes">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>201</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>201</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 81 20pt &quot;나눔스퀘어 ExtraBold&quot;;
background-color: rgb(255, 212, 222);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_dg_no">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>201</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>201</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 81 20pt &quot;나눔스퀘어 ExtraBold&quot;;
background-color: rgb(213, 232, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_dg_basic">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>201</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>201</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 81 20pt &quot;나눔스퀘어 ExtraBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="plz_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>1366</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>768</width>
     <height>1366</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>768</width>
     <height>1366</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>402</width>
     <height>316</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="dg_last_1">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="dg_last_2">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="dg_last_3">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>plz_bg</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>rec_label_5</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>rec_label_7</zorder>
  <zorder>rec_label_4</zorder>
  <zorder>rec_label</zorder>
  <zorder>rec_label_2</zorder>
  <zorder>rec_label_6</zorder>
  <zorder>dg_number</zorder>
  <zorder>dg_last_1</zorder>
  <zorder>dg_last_2</zorder>
  <zorder>dg_last_3</zorder>
 </widget>
 <resources>
  <include location="../../../../GUI/확정 ui/mainbg.qrc"/>
 </resources>
 <connections/>
</ui>
